packages/react-interactions/accessibility/README.md

```diff
@@ -14,74 +14,11 @@ can be found [here](./docs).
 
 Note: React Scopes require the internal React flag `enableScopeAPI`.
 
-When creating a scope, a query function is required. The query function is used
```

Comment:
I'm removing this document as we really don't want people to be actively using any of these APIs right now.